FAQ.md

```diff
@@ -29,4 +29,4 @@ added.
 
 ### How do I migrate from the GAIA/Kotlin version of AIF to the Java version?
 
-Refer to the [AIF v1.0.4 FAQ](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/blob/tag/v1.0.4/FAQ.md)
\ No newline at end of file
+Refer to the [AIF v1.0.5 FAQ](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/blob/tag/v1.0.5/FAQ.md)
```

Comment:
Okay, so I think I finally remembered / figured this out.  In the FAQs moving forward (starting with v1.0.5), we are linking to the v1.0.4 FAQ for instructions on upgrading from Kotlin so we don't have to keep updating the version number in this file, since it's kind of pointless because it's doubtful anybody is still upgrading from kotlin to java.  Therefore, this file should not be incremented to v1.0.5 and should be left unchanged.

Reply:
I have reverted the change to this file.